service-api/module/Application/test/ApplicationTest/Services/DWP/DwpApi/DwpApiServiceTest.php

```diff
@@ -0,0 +1,569 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApplicationTest\Services\DWP\DwpApi;
+
+use Application\DWP\AuthApi\AuthApiService;
+use Application\DWP\DwpApi\DTO\CitizenResponseDTO;
+use Application\DWP\DwpApi\DTO\DetailsResponseDTO;
+use Application\DWP\DwpApi\DwpApiException;
+use Application\Model\Entity\CaseData;
+use GuzzleHttp\Exception\ClientException;
+use Application\DWP\DwpApi\DwpApiService;
+use Application\DWP\DwpApi\DTO\CitizenRequestDTO;
+use Application\DWP\DwpApi\DTO\DetailsRequestDTO;
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response as GuzzleResponse;
+use Laminas\Http\Response;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class DwpApiServiceTest extends TestCase
+{
+    private AuthApiService&MockObject $dwpAuthApiService;
+    private DwpApiService $dwpApiService;
+    private LoggerInterface&MockObject $logger;
+    private const CASE = [
+        "id" => "b3ed53a7-9df8-4eb5-9726-abd763e6d595",
+        "personType" => "donor",
+        "lpas" => [
+            "M-XYXY-YAGA-35G3"
+        ],
+        "documentComplete" => false,
+        "identityCheckPassed" => null,
+        "yotiSessionId" => "00000000-0000-0000-0000-000000000000",
+        "idMethodIncludingNation" => [
+            "id_method" => "NATIONAL_INSURANCE_NUMBER",
+            "id_route" => "TELEPHONE",
+            "id_country" => "GBR",
+            "id_value" => "ZZ123456A"
+        ],
+        "caseProgress" => [
+            "abandonedFlow" => null,
+            "docCheck" => [
+                "idDocument" => "NATIONAL_INSURANCE_NUMBER",
+                "state" => true
+            ],
+            "kbvs" => null,
+            "fraudScore" => [
+                "decision" => "ACCEPT",
+                "score" => 265
+            ]
+        ],
+        "claimedIdentity" => [
+            "dob" => "1986-09-03",
+            "firstName" => "Lee",
+            "lastName" => "Manthrope",
+            "address" => [
+                "postcode" => "SO15 3AA",
+                "country" => "GB",
+                "line3" => "",
+                "town" => "Southamption",
+                "line2" => "",
+                "line1" => "18 BOURNE COURT"
+            ],
+            "professionalAddress" => null
+        ]
+    ];
+
+    private const DETAILS_RESPONSE = [
```

Comment:
I don't think there's any point including any of the data we don't use in this: it makes the test hard to read and makes it look like we're depending on fields which we're not.